packages/mobile/src/localCurrency/SelectLocalCurrency.tsx

```diff
@@ -0,0 +1,65 @@
+import SelectionOption from '@celo/react-components/components/SelectionOption'
+import React, { useCallback } from 'react'
+import { FlatList, StyleSheet } from 'react-native'
+import { useDispatch } from 'react-redux'
+import i18n from 'src/i18n'
+import { selectPreferredCurrency } from 'src/localCurrency/actions'
+import { LOCAL_CURRENCY_CODES, LocalCurrencyCode } from 'src/localCurrency/consts'
+import { useLocalCurrencyCode } from 'src/localCurrency/hooks'
+import { headerWithCancelButton } from 'src/navigator/Headers'
+import { navigateBack } from 'src/navigator/NavigationService'
+
+const DEFAULT_CURRENCY_CODE = 'USD'
```

Comment:
Nit, how about using your currency enum here instead of a string?

Reply:
Good catch! 👍